apache/superset
# chore: remove supersetTheme with withTheme

### SUMMARY
This pr removes the supersetTheme with withTheme component. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before

https://user-images.githubusercontent.com/17326228/136892548-0424457b-cae7-49c3-aba7-45b12990b35b.mov


after
https://user-images.githubusercontent.com/17326228/136892210-bea4877b-8166-4714-9753-64e92af57b23.mov


### TESTING INSTRUCTIONS
Go to any chart and go to the datasource panel and click on edit datasource. Check modal looks identical to original.

### ADDITIONAL INFORMATION
